clients/web/src/views/layout/FooterView.js

```diff
@@ -9,9 +9,16 @@ import 'girder/stylesheets/layout/footer.styl';
  * This view shows the footer in the layout.
  */
 var LayoutFooterView = View.extend({
+
+    initialize: function (settings) {
+        var mail = settings.contactEmail; // Has to change
```

Comment:
I used a blank ('') instead, but do you mean test if `settings.contactEmail` is provided and if not don't use href and hide the contact link ?